src/ui/public/kuery/functions/__tests__/is.js

```diff
@@ -71,39 +71,35 @@ describe('kuery functions', function () {
         expectDeepEqual(result, expected);
       });
 
-      it('should return an ES multi_match query when fieldName is "*"', function () {
-        const expected = {
-          multi_match: {
-            query: 200,
-            fields: ['*'],
-            type: 'phrase',
-            lenient: true,
-          }
-        };
-
+      it('should return a bool query with one sub-query per field when fieldName is "*"', function () {
```

Comment:
Since there is no equivalent of `multi_match` for range queries, and I want the wildcard field names to also work with range queries, I'm getting rid of the use of multi_match and using bool here so that our field name expansion will work consistently across query types. This also has the advantage of working across scripted fields as well.